src/index.ts

```diff
@@ -421,6 +423,46 @@ export const privateToAccount = function(privateKey: Buffer): Account {
   }
 }
 
+/**
+ * Returns a randomly generated mnemonic.
+ * @return {string}
+ */
+ export { generateMnemonic } from 'bip39';
+
+/**
+ * Returns an private key with the given mnemonic.
+ * @param {string} mnemonic
+ * @param {number} index
+ * @return {Buffer}
+ */
+ export const mnemonicToPrivatekey = function(mnemonic: string, index: number = 0): Buffer {
+  if (index < 0) {
+    throw new Error('[ain-util] mnemonicToPrivatekey: index should be greater than 0');
+  }
+
+  if (!validateMnemonic(mnemonic)) {
+    throw new Error('[ain-util] mnemonicToPrivatekey: Invalid mnemonic');
+  }
+
+  const seed = mnemonicToSeedSync(mnemonic);
+  const HDkey = require('hdkey');
```

Comment:
Could you move the require to the top of the file?

Reply:
Done! Thanks for comment